crdcheck-clouddriver/crdcheck-clouddriver.gradle

```diff
@@ -16,6 +16,8 @@ repositories {
     mavenCentral()
     jcenter()
     maven { url "http://dl.bintray.com/spinnaker/spinnaker/" }
+    // Used for testing against the Google product releases.
```

Comment:
this isn't true anymore, right? the plugin is compiled against this version

Reply:
that's right, I just copied the comment but now the plugins is compiled against Google version